test/e2e/exec_test.go

```diff
@@ -87,14 +87,12 @@ var _ = Describe("Podman exec", func() {
 		session := podmanTest.Podman([]string{"exec", "--env", "FOO=BAR", "test1", "printenv", "FOO"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
-		match, _ := session.GrepString("BAR")
-		Expect(match).Should(BeTrue())
+		Expect(session.OutputToString()).To(ContainSubstring("BAR"))
```

Comment:
It is better to have
 		Expect(session.OutputToString()).To(Equal("BAR"))
That way we don't match FOOBAR

Reply:
Fixed.